src/xSTIR/cSTIR/include/sirf/STIR/stir_x.h

```diff
@@ -911,13 +915,33 @@ The actual algorithm is described in
 			sptr_ad_ = sptr;
 			set_proj_data_sptr(sptr->data());
 		}
-		void set_acquisition_model(stir::shared_ptr<AcqMod3DF> sptr)
+		void set_acquisition_model(stir::shared_ptr<AcqMod3DF> sptr_am)
 		{
-			sptr_am_ = sptr;
-			AcqMod3DF& am = *sptr;
+			sptr_am_ = sptr_am;
+			AcqMod3DF& am = *sptr_am;
+			stir::shared_ptr<PETAcquisitionSensitivityModel> sptr_asm = am.asm_sptr();
 			set_projector_pair_sptr(am.projectors_sptr());
-			if (am.additive_term_sptr().get())
-				set_additive_proj_data_sptr(am.additive_term_sptr()->data());
+			bool have_a = am.additive_term_sptr().get();
```

Comment:
I'm fairly sure you don't need the `get()` (but haven't tried)

Reply:
dropping `.get()` not allowed by Visual Studio